pkg/plugin/processor/builtin/unwrap/opencdc_test.go

```diff
@@ -596,16 +590,7 @@ func TestUnwrapOpenCDC_Process(t *testing.T) {
 
 			got := underTest.Process(ctx, []opencdc.Record{tt.record})
 			is.Equal(1, len(got))
-			errRec, gotErr := got[0].(sdk.ErrorRecord)
-			if gotErr {
-				wantErr, ok := tt.want.(sdk.ErrorRecord)
-				is.True(ok)
-				is.Equal(wantErr.Error.Error(), errRec.Error.Error())
-
-				return
-			}
-
-			diff := cmp.Diff(tt.want, got[0], cmpopts.IgnoreUnexported(sdk.SingleRecord{}))
+			diff := cmp.Diff(tt.want, got[0], cmpProcessedRecordOpts...)
```

Comment:
This could be simplified a bit:
```suggestion
			is.Equal("", cmp.Diff(tt.want, got[0], cmpProcessedRecordOpts...))
```

Reply:
Nice one!